library/objective-c/Envoy.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+@interface Envoy : NSObject
+
+/// Indicates whether this Envoy instance is currently active and running.
+@property (nonatomic, readonly, getter=isRunning) BOOL running;
```

Comment:
Just realized these are already `readonly`:
```suggestion
@property (nonatomic, readonly) BOOL isRunning;
```

Reply:
Yeah I forget why I did it that other way.